src/libm-tester/gnuabi_compatibility.c

```diff
@@ -222,7 +221,6 @@ DECLARE_SP(fminf, vv);
 DECLARE_SP(fmodf, vv);
 DECLARE_SP(frfrexpf, v);
 DECLARE_SP(hypotf, vv);
-DECLARE_SP(ilogbf, v);
```

Comment:
Please don't do this, we use `ilogbf` in arm compiler for HPC. Is it possible to fix the build error? What do you mean with "it is not officially supported"?

Reply:
The problem is that the functions are called with different types of arguments from the API in gnuabi_compatiblity.c. I cannot fully explain, but the program segfaults on i386 with these two functions.

At this time, AVX version uses a data type different from AVX2 version for vint2. This is because AVX does not support 256-bit integer operations. Because of this, the functions that expose vint2 are not included in the officially supported functions.

Reply:
The API in `gnuabi_compatibility.c` is a hack that is there just to simplify the number of macros that we are using in the sources. If the problem is an AVX-only problem, then I think that `#ifdef` guard the call in the sources should be enough. Of course, this breaks a bit the claim of SLEEF being target independent, as we actually are saying that one function is not working for AVX - am I right?

Reply:
At this time, we are not claiming that those two functions work.
Anyway, I will add #ifdef to the two functions.